bin/varnishd/mgt/mgt_param.c

```diff
@@ -155,6 +155,18 @@ mcf_addpar(struct parspec *ps)
 	VTAILQ_INSERT_TAIL(&phead, pl, list);
 }
 
+static const struct parspec *
+mcf_alias(struct parspec *alias, const struct parspec *pp)
+{
+	const struct parspec *orig;
+
+	orig = TRUST_ME(pp->priv);
+	AN(orig);
+	memcpy(alias, orig, sizeof *orig);
```

Comment:
nitpick `sizeof *alias` as a "I know my buffer overflows" style checkbox?

Reply:
Fair enough, I will change the `sizeof` operand.

Should we have a `COPY_OBJ(dest, src, magic)` macro with actual type checking?